src/components/messages/conversation-preview.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { FC } from "react";
+import { Image, StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import { Conversation, Profile } from "../../models";
+
+const styles = StyleSheet.create({
+  container: {
+    height: 65,
+    justifyContent: "center",
+    marginVertical: 2,
+  },
+  infoContainer: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+    paddingHorizontal: 5,
+  },
+  picture: {
+    height: 60,
+    width: 60,
+    borderRadius: 60 / 2,
+  },
+  titleAndMessageContainer: {
+    flexDirection: "column",
+    alignItems: "flex-start",
+    justifyContent: "center",
+    width: "100%",
+    paddingLeft: 10,
+  },
+  title: {
+    fontWeight: "bold",
+    fontSize: 17,
+    marginBottom: 4,
+  },
+  messageAndTime: {
+    flexDirection: "row",
+    justifyContent: "space-between",
+  },
+  message: {
+    flex: 0.7,
+    fontSize: 14,
+    color: "#9E9E9E",
+  },
+  date: {
+    flex: 0.3,
+    fontSize: 14,
+    color: "#9E9E9E",
+    paddingLeft: 10,
+  },
+});
+
+type Props = {
+  conversation: Conversation;
+  profile: Profile;
+};
+
+export const ConversationPreview: FC<Props> = (props: Props) => {
+  const getTitle: () => string = () => {
+    if (props.conversation.title) return props.conversation.title;
+    const profiles = props.conversation.users.filter(
+      (user) => user.id !== props.profile.id
+    );
+    const fullnames = profiles.map((profile) => {
+      return [profile.firstName, profile.lastName].join(" ");
```

Comment:
Using String concatenation is more suitable for a known number of elements, as it is more efficient than Array join (https://jsben.ch/pjHHq)